homeworks/igor.orel_igor0806/GamesProject/form.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Form</title>
+        <link rel="stylesheet" href="form.css">
+    </head>
+    <body>
+
+        <!-- Header  -->
+        <header class="intro">
+            <div class="nav">
+                <div class="logo">
+                    <img src="icon/logo.svg" alt="img">
+                </div>
+
+                <div class="nav-menu">
+                    <ul class="link_list">
+
+                        <li class="link">
+                            <a class="nav-link" href="./games.html">Games</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="./index.html">About</a>
+                        </li>
+
+                        <li class="link">
+                            <a class="nav-link" href="./form.css">Login</a>
+                        </li>
+
+                    </ul>
+                </div>
+            </div>
+        </header>
+
+        <h1 class="login_title">Login page</h1>
+        <div class="form_centr">
+
+            <form class="login_form" action="./succes.html">
+                <label for="email">Email</label>
+                <br>
```

Comment:
Please remove all` <br>` tags. You can use `display: block` so that the label and input will be positioned one under the other